docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -0,0 +1,149 @@
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Schedule          | ``Schedule<Dynamic>``, ``Schedule<Static>``                               | Specify scheduling policy for work items. ``Dynamic`` scheduling is implemented through a work stealing queue. Default is machine and backend specific. |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| IndexType         | ``IndexType<int>``                                                        | Specify integer type to be used for traversing the iteration space. Defaults to ``int64_t``.                                                            |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| LaunchBounds      | ``LaunchBounds<MaxThreads, MinBlocks>``                                   | Specifies hints to to the compiler about CUDA/HIP launch bounds.                                                                                        |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| WorkTag           | ``SomeClass``                                                             | Specify the work tag type used to call the functor operator. Any arbitrary type defaults to ``void``.                                                   |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Requirements
+~~~~~~~~~~~~
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+ 
+.. code-block:: cpp
+    
+    RangePolicy()
+    
+\
+    Default Constructor uninitialized policy.
+
+.. code-block:: cpp
+
+    template<class ... InitArgs> 
+    RangePolicy(const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+
+\
+    Provide a start and end index as well as optional arguments to control certain behavior (see below).
+   
+.. code-block:: cpp
+
+    template<class ... InitArgs> 
+    RangePolicy(const ExecutionSpace& space, const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
```

Comment:
why code block instead of cp::function?

Reply:
For the same reasons as here: https://github.com/kokkos/kokkos-core-wiki/pull/230#issuecomment-1364142052
It is the presence of the three small dots that is not understood by the sphinx syntax.